packages/react-core/src/components/EmptyState/EmptyStateIcon.tsx

```diff
@@ -1,39 +1,31 @@
 import * as React from 'react';
 import { css } from '@patternfly/react-styles';
 import styles from '@patternfly/react-styles/css/components/EmptyState/empty-state';
+import { Spinner } from '../Spinner';
 
 export interface IconProps extends Omit<React.HTMLProps<SVGElement>, 'size'> {
   /** Changes the color of the icon.  */
   color?: string;
 }
 
 export interface EmptyStateIconProps extends IconProps {
-  /** Additional classes added to the EmptyState */
+  /** Additional classes added to the EmptyStateIcon */
   className?: string;
   /** Icon component to be rendered inside the EmptyState on icon variant
    * Usually a CheckCircleIcon, ExclamationCircleIcon, LockIcon, PlusCircleIcon, RocketIcon
-   * SearchIcon, or WrenchIcon */
-  icon?: React.ComponentType<any>;
-  /** Component to be rendered inside the EmptyState on container variant */
-  component?: React.ComponentType<any>;
-  /** Adds empty state icon variant styles  */
-  variant?: 'icon' | 'container';
+   * SearchIcon, or WrenchIcon. Can be a Spinner component too.*/
```

Comment:
We could use a similar description as the `icon` prop in EmptyStateHeader:

```suggestion
   * Icon component to be rendered. Can also be a Spinner component. */
```